Dockerfile

```diff
@@ -1,17 +1,17 @@
-FROM golang:1.12.4 as daemon
+FROM golang:1.12.9 as daemon
 
 COPY . /go/src
 WORKDIR /go/src
 RUN env GO111MODULE=on go build
 
-FROM node:10.15.3 as gui
+FROM node:10.16.3 as gui
```

Comment:
Is 11.15 available?

Reply:
As per matrix chat, current (12.x) is preferable for smoking out issues we'll eventually have to face.

Reply:
travis builds the Dockerfile, which will test LTS.
travis also builds using the decred/dcrdata-golang-builder container which has current (12.x)